cernopendata/modules/fixtures/data/records/data-policies.json

```diff
@@ -350,4 +350,48 @@
       ]
     }
   }
+  {
+    "abstract": {
+      "description": "This document defines the DELPHI data preservation, re-use, and open access policy as endorsed by the collaboration board in spring 2024."
+    },
+    "accelerator": "CERN LEP",
+    "authors": [
+      {
+        "name": "The DELPHI collaboration"
+      }
+    ],
+    "collections": [
+      "Data-Policies"
+    ],
+    "date_published": "2024",
+    "distribution": {
+      "formats": [
+        "pdf"
+      ]
+    },
+    "experiment": [
+      "DELPHI"
+    ],
+    "files": [
+      {
+        "checksum": "adler32:cf535436",
+        "size": 45811,
+        "uri": "root://eospublic.cern.ch//eos/opendata/delphi/documentation/DELPHI_Data_preservation-8.pdf"
+      }
+    ],
+    "language": "English",
+    "prepublication": {
+      "date": "2024-06-26",
```

Comment:
Should we say 2024-03-12 here so that the prepublication date would correspond to the date inscribed on the PDF document of the policy?